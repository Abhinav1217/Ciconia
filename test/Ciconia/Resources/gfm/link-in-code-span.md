Look at: `http://someurl.com` and text.