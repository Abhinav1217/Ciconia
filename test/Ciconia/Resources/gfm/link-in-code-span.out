<p>Look at: <code>http://someurl.com</code> and text.</p>